libbeat/idxmgmt/ilm/ilm_test.go

```diff
@@ -220,9 +220,11 @@ func TestDefaultSupport_Manager_EnsurePolicy(t *testing.T) {
 		calls     []onCall
 		overwrite bool
 		cfg       map[string]interface{}
+		create    bool
 		fail      error
 	}{
 		"create new policy": {
+			create: true,
```

Comment:
I'm n00b here, so probably this is not an issue but a style preference

wouldn't all expected vars be better named with `expect` prefix? like `expectCreated` or `expectFail`?

dismiss if you don't think so

Reply:
Rather have something named `must`. This is a hot fix and I will create a follow up for testing.

Reply:
sure, sorry for adding noise. Not an issue